oemoflex/component_attrs/shortage.csv

```diff
@@ -1,11 +1,11 @@
 attribute,type,unit,default,suffix,description
 region,str,n/a,n/a,,Country code
-name,str,n/a,,-el-shortage,Name of the bus
+name,str,n/a,,-el-shortage,Name
 type,str,n/a,dispatchable,,Type
-carrier,str,n/a,shortage,,
-tech,str,n/a,shortage,,
-bus,str,n/a,,-el-bus,
-capacity,float,MW,,,
-marginal_cost,float,Eur/kWh,n/a,,
-profile,str,n/a,1,,
-output_parameters,dict,n/a,{},,
+carrier,str,n/a,shortage,,Energy carrier
+tech,str,n/a,shortage,,Technology
+bus,str,n/a,,-el-bus,The bus this component is connected to
+capacity,float,MW,,,Installed capacity
+marginal_cost,float,Eur/MWh,n/a,,Marginal cost
+profile,str,n/a,1,,Name of the profile
```

Comment:
Should the unit be dropped here, since `profile`'s value  is only a reference? Or should it specify the unit of the referenced timeseries?

Reply:
It could specify the unit of the profile. However, the profile itself is unitless. It is multiplied by  `amount`(for load) or `capacity` (for dispatchables), respectively.

Reply:
Shortage should not have a profile anyway. Dropped it.